### DISCUSSION TITLE
Can I create a new viewpoint to send video from that view point to remote sensor/systems?

### DISCUSSION THREAD

**Discussion by robogrow** [microsoft/AirSim#4317] [Support Q&A]
I am working on a smart crosswalk study. We have start to capture 3D meshes of the road network.

Can AirSim allow me talk a FOV video stream and send it to vendors AI camera for testing and performance evaluation?

I'd like to generate lots of different pedestrian, bike and traffic scenarios under different lighting and on different roadways to automate the testing of around 30 vendor systems..

I see there is a ground view, 3D person view and first person view. How can a create new views. It would be nice to create one view and stream it to many Ai vision systems to compare them side by side. Several of the vendors have fisheye lenses. It there an easy way to create a fisheye lens view or do I need to write a filter that morphs a regular video into a fisheye view? peter@ccaway.net

**Comment by alonfaraj** [comment]
Maybe External Camera can fits to your needs.  
You can find more details [here](https://microsoft.github.io/AirSim/settings/#external-cameras).  
AirSim camera support distortion model, so if you set a wide FOV with your distortion params, it might work, not sure.
